sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/executable/SonarTSCoreBundle.java

```diff
@@ -87,16 +95,23 @@ private void deploy() {
    * Builds command to run rules and calculate metrics with tsrunner
    */
   @Override
-  public SonarTSRunnerCommand getSonarTsRunnerCommand(String tsconfigPath, Iterable<InputFile> inputFiles, TypeScriptRules typeScriptRules) {
+  public SonarTSCommand getSonarTsRunnerCommand() {
+    return getCommand(sonartsRunnerExecutable);
+  }
+
+  @Override
+  public String getRequestForRunner(String tsconfigPath, Iterable<InputFile> inputFiles, TypeScriptRules typeScriptRules) {
+    SonarTSRequest request = new SonarTSRequest();
+    request.filepaths = StreamSupport.stream(inputFiles.spliterator(), false).map(InputFile::absolutePath).toArray(String[]::new);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "absolutePath"; it is deprecated. ([squid:CallToDeprecatedMethod](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=sonarts&branch=issue-351&issues=AWD5U5ejEFfjxkQcPdcn&open=AWD5U5ejEFfjxkQcPdcn)

[ISSUE_KEY]: AWD5U5ejEFfjxkQcPdcn